lib/auth/auth.js

```diff
@@ -111,11 +113,14 @@ function doAuth(request, log, cb, awsService, requestContexts) {
         return cb(null, res.params);
     }
 
-    requestContexts.forEach(requestContext => {
-        requestContext.setAuthType(res.params.data.authType);
-        requestContext.setSignatureVersion(res.params.data.signatureVersion);
-        requestContext.setSignatureAge(res.params.data.signatureAge);
-    });
+    if (requestContexts) {
```

Comment:
For batch delete, we authenticate first without checking policies.  This means that we don't have to parse the xml sent with the request before authenticating.  Once we have authenticated successfully, we send a request to vault to just authorize against the policies if the request is from a user.